### PR TITLE
Bookmark

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
image

### CODE REVIEW

File: src/components/posts/Posts.js [hunk 1]
```diff
@@ -71,6 +72,20 @@ export const Posts = ({
     );
     setCommentText("");
   };
+
+  const AddToBookMarkHandler = ()=>{
+    console.log("clicked")
```
Comment (Muzammil-khan710) [inline src/components/posts/Posts.js]: remove console

File: src/components/posts/Posts.js [hunk 2]
```diff
@@ -71,6 +72,20 @@ export const Posts = ({
...
+    dispatch(addToBookMark({token:encodedToken, postId:_id}))
+    setIsPostBookmarked(!isPostBookmarked)
```
Comment (Rockyraven) [inline src/components/posts/Posts.js:78]: Remove this console

File: src/redux/slices/bookmarkSlice.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+export const fetchBookMark = createAsyncThunk("/bookmark/fetch", async (thunkAPI)=>{
+
+    try {
+        const res = await axios.get("",{})
+        console.log(res)
+    } catch (error) {
+       return  thunkAPI.rejectedWithValue(error)
+    }
```
Comment (Rockyraven) [inline src/redux/slices/bookmarkSlice.js:18]: Remove this console also

**Review by Rockyraven — APPROVED** [review]
Remove unwanted console 
Otherwise your code is clean and easy to readable
You can merge

**Review by Muzammil-khan710 — APPROVED** [review]
Remove console.log
and after that it is good to merge

**Pull request merged**